### DISCUSSION TITLE
Ability to track crypto transactions and wallet addresses.

### DISCUSSION THREAD

**Discussion by GrandArcher** [serversideup/financial-freedom#112] [Ideas]
# 👉 Describe the problem
[comment]: <> (What are you trying to solve?) 
Ability to track my crypto addresses.

# 👥 Problem evidence & reach
[comment]: <> (How many people have this problem?) 
I can only make and assumption here but I would assume a lot of people would have the same desire.

# 🥰 Describe the "impact" on users?
[comment]: <> (How will this make people's lives better once it is solved?) 
Ability to centralize , track, and tag crypto transactions.

# 🏆 How to solve this problem
[comment]: <> (Describe the feature that you are proposing and how it will solve the problem) 

I would like to see the ability to add crypto accounts that can be tracked by wallet address(s) per account. 
I would like the ability to tag transactions.

Honestly would be happy if it was just bitcoin but others would be nice as I know this is a complicated feature request (long term). (Ethereum, polygon, doge, solana, tezos... Ect..)

**Comment by danpastori** [comment]
Definitely a great idea! Once we re-build the pillars of the app (accounts, transactions, budgets), I'll write up some expansion documentation where we can quickly make plugins to feed data into Financial Freedom. Like, create an interface and bring your own data.

Do you have any APIs that you prefer to work with for Crypto? I've used Polygon [https://polygon.io/](https://polygon.io/) for tracking prices and it's super slick.
